ppl/detail/ZeekTag.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from "classnames"
+import React from "react"
+import styled from "styled-components"
+import {BrimEventInterface} from "./models/BrimEvent"
+
+const BG = styled.div`
+  height: 18px;
+  border-radius: 3px;
+  display: inline-flex;
+  color: white;
+  cursor: default;
+  user-select: none;
+  justify-content: flex-end;
```

Comment:
Just gonna throw out that I personally prefer the justification to be centered here, looks a little neater that way imo

Reply:
You got it.